.pre-commit-config.yaml

```diff
@@ -23,27 +23,32 @@ repos:
     rev: v4.4.0
     hooks:
       - id: check-added-large-files
-        args: [ --maxkb=64 ]
       - id: check-executables-have-shebangs
       - id: check-shebang-scripts-are-executable
-      - id: check-symlinks
       - id: check-toml
-      - id: check-vcs-permalinks
       - id: check-yaml
       - id: end-of-file-fixer
       - id: mixed-line-ending
-        args: [ --fix=no ]
```

Comment:
I don't think we want it to fix on pre-commit, right?

Reply:
This was copied from tpaw, I left it as is cause I wasn't sure of the implications of changing line ending of files that might have different ones.

Reply:
Oh I read that wrong. I'll put it back.